MAUI/Calendar/customizations.md

```diff
@@ -0,0 +1,257 @@
+            {
+                return false;
+            }
+
+            return true;
+        };
+
+        this.Calendar.MonthView.SpecialDates = new List<DateTime>
+        {
+            DateTime.Now.AddDays(2),
+            DateTime.Now.AddDays(-2),
+        };
+
+        this.Calendar.MonthView.WeekendDays = new List<DayOfWeek>
+        {
+            DayOfWeek.Sunday,
+            DayOfWeek.Saturday,
+        };
+
+        this.Calendar.Background = Colors.PaleGreen;
+        this.Calendar.MonthView.TextStyle = textStyle;
+        this.Calendar.MonthView.TodayBackground = Colors.Pink;
+        this.Calendar.MonthView.TodayTextStyle = textStyle;
+        this.Calendar.ShowTrailingAndLeadingDates = true;
+        this.Calendar.MonthView.TrailingLeadingDatesBackground = Colors.Green;
+        this.Calendar.MonthView.TrailingLeadingDatesTextStyle = textStyle;
+        this.Calendar.MonthView.DisabledDatesBackground = Colors.Grey;
+        this.Calendar.MonthView.DisabledDatesTextStyle = textStyle;
+        this.Calendar.MonthView.SpecialDatesBackground = Colors.Purple;
+        this.Calendar.MonthView.SpecialDatesTextStyle = textStyle;
+        this.Calendar.MonthView.WeekendDatesBackground = Colors.Red;
+        this.Calendar.MonthView.WeekendDatesTextStyle = textStyle;
+
+{% endhighlight %}
+{% endtabs %}
+
+![monthview-customization-in-maui-calendar](images/customization/monthview-customization-in-maui-calendar.png)
+
+>**NOTE**
+* The Background color and text style will be applied based on the following order: selectableDayPredicate dates, special dates, disable dates, today date, trailingLeading Dates and weekend dates.
+
+## Month View Header format
+You can customize the month format of the `Calendar` by using the `TextFormat` property.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<calendar:SfCalendar  x:Name="Calendar" 
+                      View="Month">
+            <Calendar:SfCalendar.MonthView>
+                <Calendar:CalendarMonthView>
+                    <Calendar:CalendarMonthView.HeaderView>
+                        <Calendar:CalendarMonthHeaderView TextFormat="ddd" />
+                    </Calendar:CalendarMonthView.HeaderView>
+                </Calendar:CalendarMonthView>
+            </Calendar:SfCalendar.MonthView>
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+this.Calendar.MonthView.HeaderView = new CalendarMonthHeaderView()
+{
+    TextFormat = "ddd",
+};
+
+{% endhighlight %}
+{% endtabs %}
+
+![view-header-text-format-in-maui-calendar](images/customization/view-header-text-format-in-maui-calendar.png)
+
+## Year cell customization
+You can customize the calendar `year`, `decade` and `century` view by using the `YearView` property of `SfCalendar`. 
```

Comment:
You can customize the calendar `year`, `decade,` and `century` view by using the `YearView` property of `SfCalendar.`

Reply:
Content changed